### PR TITLE
Add code review policy

### PR DESCRIPTION
**Pull request by Dschoordsch**

### CODE REVIEW

File: docs/codeReview.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    e.g. change will crash, high impact performance, high impact on maintainability
+    - -1 please fix this
+    e.g. naming of variable is misleading, coupling of components can be reduced, code is hard to understand
+    - +1 I would do it differently
+      - inconsequential enough to not need another review
+      - some suggestion for the authors consideration, e.g. better variable name, alternative split of components
+      - refactoring of existing code, e.g. While you're in here, could you fix xyz?
+    - +2 kudos
+    e.g. nice work here, I learnt something, good find
+    - refactor? ie. the author's doing things in a part of the code that could be improved (for example changing from fragment containers to relay hooks). Then it’s the author’s responsibility to make a decision to either fix it (if it’s simple enough like 10 min work, or create a separate task from a given comment)
```
[the remark this answers is not in the document]
Comment (Dschoordsch) [reply docs/codeReview.md:19]: It's not official yet, but you're right, I tried to incorporate this in line 16. If you're fine with it I would remove this bullet point

[the remark this answers is not in the document]
Comment (BartoszJarocki) [reply docs/codeReview.md:19]: sure, it turned out numbers work great for us so let's get rid of this part. really good idea to have it here!

**Review by BartoszJarocki — APPROVED** [review]

File: docs/codeReview.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      - inconsequential enough to not need another review
+      - some suggestion for the authors consideration, e.g. better variable name, alternative split of components
+      - refactoring of existing code, e.g. While you're in here, could you fix xyz?
+    - +2 kudos
+    e.g. nice work here, I learnt something, good find
+- Final review is approval if there are no negative comments, otherwise change requested
+    - Changes requested? Move ticket to self-review
+    - Approved? Move to maintainer-review
+    - Approved by the maintainer
+      - lowest score +1? the PR goes back to self-review
```
Comment (nickoferrall) [inline docs/codeReview.md:23]: +1 I'd avoid using the word score as it implies the numbers are being summed. Perhaps, "if there are no -2 or -1 comments, but there are +1 comments, the PR goes back to self-review"?

On the line below, we could say "if there are no comments or only +2 comments, the PR gets merged"?

**Review by nickoferrall — APPROVED** [review]
LGTM! Thanks for writing this up

File: docs/codeReview.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Code review policy
+
+The goal is to help reviewers to make their intentions clear to the author and for the author to know exactly what is expected of to pass the next review round.
```
Comment (enriquesanchez) [inline docs/codeReview.md:3]: `...what is expected of to pass...`

Should it say instead "what is expected of _them_ to pass"?

File: docs/codeReview.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    - -1 please fix this
+    e.g. naming of variable is misleading, coupling of components can be reduced, code is hard to understand
+    - +1 I would do it differently
+      - inconsequential enough to not need another review
+      - some suggestion for the authors consideration, e.g. better variable name, alternative split of components
+      - refactoring of existing code, e.g. While you're in here, could you fix xyz?
+    - +2 kudos
+    e.g. nice work here, I learnt something, good find
+- Final review is approval if there are no negative comments, otherwise change requested
+    - Changes requested? Move ticket to self-review
```
Comment (tianrunhe) [inline docs/codeReview.md:20]: \-1 Can we be clear here that by "ticket" we mean the issue this PR is corresponding to.

File: docs/codeReview.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    - Approved? Move to maintainer-review
```
Comment (tianrunhe) [inline docs/codeReview.md:21]: \-1 "Move the issue to maintainer-review"

**Review by tianrunhe — CHANGES_REQUESTED** [review]

**Comment by tianrunhe** [comment]
The policy looks good for us internally. My request is more on behalf of external users: let's make it more explicit for some terms we use here.

**Review by tianrunhe — APPROVED** [review]
Great work! Thanks for creating this policy!

**Comment by mattkrick** [comment]
love it! we need this to live in github for external PR contributions. in the future, we will move governance to its own github repo, but that's a future problem & this does a good job of building the case for moving our governance from notion to GH.

**Review by mattkrick — APPROVED** [review]

**Review by enriquesanchez — APPROVED** [review]

**Comment by mattkrick** [comment]
that's a whole lot of green checkmarks. merging!

**Pull request merged**